sdk/search/search/README.md

```diff
@@ -177,7 +189,7 @@ const ratingMin = 4;
 const iterator = client.listSearchResults({
```

Comment:
Still not crazy about this name. Just `search` is too far from being idiomatic?

Reply:
I dislike the name as well as the pattern (exposing server paging to the developer.) This is something we need to discuss more, so I think it might slip to preview 2 (though we'll have it in nightly before then)

Reply:
updated with the better name/pattern we discussed